src/ssl.c

```diff
@@ -7631,6 +7631,18 @@ int wolfSSL_use_PrivateKey_ASN1(int pri, WOLFSSL* ssl, unsigned char* der,
     return wolfSSL_use_PrivateKey_buffer(ssl, der, derSz, WOLFSSL_FILETYPE_ASN1);
 }
 
+int wolfSSL_CTX_use_PrivateKey_ASN1(int pri, WOLFSSL_CTX* ctx,
```

Comment:
Could use a brief comment section above some of the additional API added. Looks like some got it but a few were missed.